mmselfsup/models/algorithms/base.py

```diff
@@ -62,6 +63,9 @@ def __init__(self,
         if head is not None:
             self.head = MODELS.build(head)
 
```

Comment:
We can add some comments here to illustrate the goal of target_generator, to help the code reading.